app-dicts/myspell-he/myspell-he-20120815.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MYSPELL_DICT=(
+	"he_IL.aff"
+	"he_IL.dic"
+)
+
+inherit myspell-r2
+
+DESCRIPTION="Hebrew dictionaries for myspell/hunspell"
+HOMEPAGE="https://extensions.libreoffice.org/extensions/hebrew-he-spell-check-dictionary"
+SRC_URI="https://extensions.libreoffice.org/assets/downloads/z/dict-he-2012-08-15.oxt"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
```

Comment:
What's the point of this ebuild? It's just differently named from what we already have in the tree.

Reply:
The link in SRC_URI is easier to read (it used to be https://extensions.libreoffice.org/extensions/hebrew-he-spell-check-dictionary/${PV//./-}/@@download/file/dict-he-${PV//./-}.oxt)

Reply:
but why are you introducing a new version just to rename the `SRC_URI`? And by our standards, the one in tree is more maintainable than hardcoding versions each version bump
(I suggest you just bump the EAPI if you need to touch this)